modules/task.js

```diff
@@ -0,0 +1,96 @@
+export default class Task {
+  constructor() {
+    this.lists = JSON.parse(localStorage.getItem('list')) || [];
+    this.editId = null;
+  }
+
+  addList() {
+    const taskInput = document.getElementById('task-input');
+    const list = { description: '', completed: false, index: 0 };
+    list.description = taskInput.value;
+    if (!list.description) {
+      return null;
+    } if (this.editId) {
+      const index = this.editId[this.editId.length - 1];
+      this.lists.splice(index, 1, list);
+      localStorage.setItem('list', JSON.stringify(this.lists));
+    } else {
+      list.index = this.lists.length + 1;
+      this.lists.push(list);
+      localStorage.setItem('list', JSON.stringify(this.lists));
+      this.showLists([list]);
+      taskInput.value = '';
+    }
+    window.location.reload();
+    this.isEditing = null;
+    return true;
+  }
+
+  showLists(list) {
+    const renderedList = list || this.lists;
+    renderedList.forEach((item, index) => {
+      const wrapper = document.getElementById('list-ul');
+      wrapper.innerHTML += `<li class="list-li" id="collection-${index}">
+      <div class="list-input">
+      <div class="input-box">
+      <input type="checkbox" class="completed" required ${
+  item.completed ? 'checked' : ''
+}>
+      <label class="check ${
+  item.completed ? 'checked' : ''
+}" id="description-${index}">${item.description}</label>
+      </div>
+      <span><i class="fa fa-trash-o del-btn btn-${index}" aria-hidden="true"></i></i></span>
+      </div>
+      </li>`;
+    });
+  }
+
+  editTask(id, description) {
+    const input = document.getElementById('task-input');
+    input.value = description;
+    this.editId = id;
+  }
+
+  delete(deleteClass) {
```

Comment:
The function name should properly convey the task it is doing. delete is a broad word. Kindly use a more specific name.